src/libraries/Microsoft.XmlSerializer.Generator/src/Sgen.cs

```diff
@@ -309,7 +309,7 @@ private void GenerateFile(List<string> typeNames, string assemblyName, bool prox
                     //However when a path ending with backslash, if followed by double quote, it becomes an escapte sequence
                     //e.g. "obj\Debug\netcoreapp2.0\", it will be converted as obj\Debug\netcoreapp2.0", which is not valid and not exist
                     //We need remove the ending quote for this situation
-                    if (!outputDirectory.EndsWith("\"") || !Directory.Exists(outputDirectory = outputDirectory.Remove(outputDirectory.Length - 1)))
+                    if (!outputDirectory.EndsWith("\"", StringComparison.Ordinal) || !Directory.Exists(outputDirectory = outputDirectory.Remove(outputDirectory.Length - 1)))
```

Comment:
Could this just use `EndsWith('\')`?

Reply:
Doesn't exist in netstandard2.0, which this project targets. :(